### PR TITLE
#612 fix dups

### PR DESCRIPTION
**Pull request by akashbilgi**
1) fixed multiple calls to sanitize ( **resanitize issue for "saved" question** )
- this also prevent  going thru al the item in answers.json and printing it on lterminal/logger 
- the answers are already saved by sanitizing - it doesnt need resanitizeing when looping through

Example

```
>  for item in self.all_data:
>                 if **self._sanitize_text(question_text)** in item['question'] and item['type'] == 'radio':
>                     existing_answer = item
```
this called sanitize every question that was ever saved and prited the question on output/logger

2) (**resanitize issue of LinkedIn question**) is fixed

3) **reloaded answers** from json once they are saved so we can get new asnwers when needed

4) Moreover all the answers were saved **(duplicated)** in many scenarios when the question/answer are presetn - the if statement was fixed to allowed only non duplicates

fixes issue #612 

[#612 was moved to completed by automation - please reopen and we can move it complete once this PR is mereged]

### CODE REVIEW

**Comment by akashbilgi** [comment]
@feder-cr 
@surapuramakhil 

please review and merge

**Comment by cjbbb** [comment]
@akashbilgi reopened

**Comment by sarob** [comment]
Happy to review after conflicts resolved

**Comment by akashbilgi** [comment]
@sarob resolved conflicts, please take a look

i think the code could be further improved by improving the sanitize func ( it is printing every question "ever"  in "every" call made ) - we can do that for next part but for stability/conservative purposes this code should be more than good ..

**Review by sarob — APPROVED** [review]
passed all test and works as expected. 

thanks!

File: src/ai_hawk/linkedIn_easy_applier.py [hunk 1]
```diff
@@ -692,7 +694,13 @@ def _find_and_handle_radio_question(self, section: WebElement) -> bool:
             question_text = section.text.lower()
             options = [radio.text.lower() for radio in radios]
 
-            existing_answer = self._find_existing_answer(question_text)
+            existing_answer = None
```
Comment (surapuramakhil) [inline src/ai_hawk/linkedIn_easy_applier.py:697]: why is this going reverse of clean code?

Comment (akashbilgi) [reply src/ai_hawk/linkedIn_easy_applier.py:697]: _find_existing_answer(question_text) - is printing unnecessary log - for every iteration( question ) that is present in answers.json

**Pull request merged**